src/toil_vg/vg_evaluation_pipeline.py

```diff
@@ -204,6 +216,142 @@ def run_pipeline_merge_vcf(job, options, vcf_tbi_file_id_pair_list):
     write_to_store(job, options, vcf_file, use_out_store = True, out_store_key = out_store_key)
     write_to_store(job, options, vcf_file_idx, use_out_store = True, out_store_key = out_store_key + ".tbi") 
 
+def generate_config():
```

Comment:
@glennhickey My proposed schema for the config file.
Initially my plan is that the main pipeline script will be able to generate and run it's own config file. Once that functionality is all there and working, I'll move on to writing config file generation and use in the component scripts (vg_index, vg_map, vg_call)